### PR TITLE
Update staff log, group member lists, and unique titles listing

### PR DESCRIPTION
**Pull request by hikikuman**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

Update `Staff Log` and BN/GMT/NAT/ALM/Curators/Users with unique titles listing, Add Afrikaans language to `group-info`

Updated list:
- Beatmap Nominator
- Global Moderator Team
- Nomination Assessment Team
- osu! Alumni
- Beatmap Spotlight Curators
- Users with unique titles

Add Afrikaans language to `group-info` for Zuid-Afrikaan (Probationary BN).

### CODE REVIEW

**Comment by cl8n** [comment]
please branch from a more recent commit on master next time ...

**Review by cl8n — CHANGES_REQUESTED** [review]
all good, but log the spotlights group as well

**Comment by hikikuman** [comment]
> all good, but log the spotlights group as well

add them all or just the recent one?

**Comment by Walavouchey** [comment]
pretty sure that means to link the spotlight curator group page (https://osu.ppy.sh/groups/48) in the spotlights article

File: wiki/People/The_Team/Beatmap_Nominators/en.md [hunk 1]
```diff
@@ -49,62 +49,58 @@ The tables listed below indicate the game mode(s) in which each Beatmap Nominato
 | Name | Additional languages |
 | :-- | :-- |
 | ::{ flag=PH }:: [-Aqua](https://osu.ppy.sh/users/7150015) | Filipino |
-| ::{ flag=US }:: [-White](https://osu.ppy.sh/users/16276548) |  |
-| ::{ flag=GB }:: [Aistre](https://osu.ppy.sh/users/4879380) |  |
+| ::{ flag=IT }:: [-Syncro](https://osu.ppy.sh/users/4338923) | Italian |
 | ::{ flag=GB }:: [AJT](https://osu.ppy.sh/users/3181083) |  |
```
Comment (Hiviexd) [inline wiki/People/The_Team/Beatmap_Nominators/en.md:53]: resigned, remove

File: wiki/People/The_Team/Beatmap_Nominators/en.md [hunk 2]
```diff
@@ -49,62 +49,58 @@ The tables listed below indicate the game mode(s) in which each Beatmap Nominato
...
 | ::{ flag=HK }:: [Saggin](https://osu.ppy.sh/users/12464107) | Chinese, Cantonese, some Japanese |
 | ::{ flag=FR }:: [Sajinn](https://osu.ppy.sh/users/13513687) | French |
 | ::{ flag=RU }:: [Shmiklak](https://osu.ppy.sh/users/5504231) | Russian |
 | ::{ flag=RU }:: [SMOKELIND](https://osu.ppy.sh/users/9327302) | Russian |
-| ::{ flag=DE }:: [Sylvarus](https://osu.ppy.sh/users/4505918) | German |
+| ::{ flag=FR }:: [Sotarks](https://osu.ppy.sh/users/4452992) | French |
 | ::{ flag=FI }:: [TheKingHenry](https://osu.ppy.sh/users/5128277) | Finnish |
 | ::{ flag=TH }:: [timemon](https://osu.ppy.sh/users/2072005) | Thai |
 | ::{ flag=LT }:: [tomatas95](https://osu.ppy.sh/users/11403815) | Lithuanian |
 | ::{ flag=CL }:: [Uberzolik](https://osu.ppy.sh/users/1314547) | Spanish, French |
```
Comment (Hiviexd) [inline wiki/People/The_Team/Beatmap_Nominators/en.md:96]: kicked, remove

File: wiki/People/The_Team/Beatmap_Nominators/en.md [hunk 3]
```diff
@@ -125,28 +122,30 @@ The tables listed below indicate the game mode(s) in which each Beatmap Nominato
 | ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) | Arabic, French, some Italian |
```
Comment (Hiviexd) [inline wiki/People/The_Team/Beatmap_Nominators/en.md:122]: moved to NAT alongside Dusk-, both should be removed from here

**Review by Hiviexd — CHANGES_REQUESTED** [review]
posted my review on `en.md` files but should adjust other languages accordingly

should also remove AJT from Elite Nominators `Users_with_unique_titles`, title gets removed when you're no longer a BN and other non-BN elite nominators aren't on the page

**Comment by hikikuman** [comment]
> > add them all or just the recent one?
> 
> all of the aiess messages for that group (should be its whole history, it looks like it logged the group from the start)

alright. i've also asked venix about that. double checked.

**Comment by cl8n** [comment]
> please branch from a more recent commit on master next time ...

and be more careful with git in general o_O no idea what you were trying to do with the merge into your old commit or removing some CI scripts

**Review by cl8n — APPROVED** [review]

**Comment by cl8n** [comment]
thanks for update, probably took forever to go through all of that

**Pull request merged**